exercises/concept/beers/tests/Tests.elm

```diff
@@ -0,0 +1,352 @@
+module Tests exposing (tests)
+
+import Beers exposing (beerSongFrom)
+import Expect
+import Test exposing (..)
+
+
+tests : Test
+tests =
+    describe "99 Bottles of Beer"
+        [ test "No bottles" <|
+            \_ ->
+                beerSongFrom 0
+                    |> Expect.equal txtBeers.none
+        , skip <|
+            test "1 bottle" <|
+                \_ ->
+                    beerSongFrom 1
+                        |> Expect.equal txtBeers.one
+        , skip <|
+            test "2 bottles" <|
+                \_ ->
+                    beerSongFrom 2
+                        |> Expect.equal txtBeers.two
+        , skip <|
+            test "99 bottles" <|
+                \_ ->
+                    beerSongFrom 99
```

Comment:
I don't think there is a need for the function to test the 99 case, it makes this file unnecessarily long! Maybe the >30 case as that is special ...